Src/BootCamp.Chapter/Logger/ConsoleLogger.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace BootCamp.Chapter.Logger
+{
+    class ConsoleLogger : ILogger
+    {
+        public void Log(string msg)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'Log' a static method.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5147695)

Reply:
Request help please.  I don't understand this.  How do I define static members on an interface?

Reply:
Ignore that bit. I disagree with codacy here. You can have a static log but then you would need non-static implementation of it as well. Just bad. 

Carry on :)

Reply:
Thank you.  I will submit and see what have to say from the review.